src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx

```diff
@@ -406,6 +412,7 @@ function PopoverReportActionContextMenu({ref}: PopoverReportActionContextMenuPro
         onConfirmDeleteModal.current = onConfirm;
         reportIDRef.current = reportID;
         reportActionRef.current = reportAction ?? null;
+        setChildReportID(reportAction?.childReportID);
```

Comment:
I don't think this is a good strategy overall. ChildReportID does not impact UI so keeping them in state is not a good idea. Can you add logic in a way which finds the necessary actions where `deleteAppReport` is called instead.